wp-content/themes/reconasia/template-parts/block-post.php

```diff
@@ -12,16 +12,26 @@
  */
 
 ?>
-
 <article <?php post_class('post-block post-block--post'); ?> id="post-<?php the_ID(); ?>">
+	<div class="post-block-right">
+		<?php
+			// if ( !is_front_page() ){
+				// echo get_the_post_thumbnail( get_the_ID(), 'large' );		
+			// }
+			echo get_the_post_thumbnail();
+		?>	
+	</div>
 
-	<?php
-
-	the_title( '<h3 class="post-block__title"><a href="' . esc_url( get_permalink() ) . '">', '</a></h3>' );
-
-	the_excerpt();
-
-	reconasia_authors();
-	?>
+	<div class="post-block-left">
+		<?php
+			the_title( '<h3 class="post-block__title"><a href="' . esc_url( get_permalink() ) . '">', '</a></h3>' );
+			
+			the_excerpt();
+			
+			reconasia_authors();
+			reconasia_posted_on();
```

Comment:
Let's use the [WP date format](https://wordpress.org/support/article/formatting-date-and-time/) to format the date correctly.  The function to call the posted on date accepts the format string, so you can do something like `reconasia_posted_on('M j, Y');`